extensions/google-drive/src/utils.ts

```diff
@@ -186,10 +153,21 @@ ${file.fileSizeFormatted}
 ---
 
 **Created**\n
-${file.createdAt.toLocaleString()}
+${new Date(file.createdAt).toLocaleString()}
```

Comment:
@daniel-stoneuk In ref to this - https://github.com/raycast/extensions/pull/1613/files#r868024738, the earlier version `file.createdAt.toLocaleString()` was printing the datetime in the user's locale, AFAIK.